### PR TITLE
Add more tests

### LINKED ISSUE #17: Add more tests

### PR DESCRIPTION
**Pull request by CaroFG**
Closes #17

### CODE REVIEW

File: lib/meilisearch-rails.rb [hunk 1]
```diff
@@ -887,7 +887,7 @@ def ms_configurations
 
     def ms_primary_key_method(options = nil)
       options ||= meilisearch_options
-      options[:id] || options[:primary_key] || :id
+      options[:primary_key] || options[:id] || :id
```
Comment (curquiza) [inline lib/meilisearch-rails.rb:890]: For what this fix is for? :)

File: spec/integration_spec.rb [hunk 2]
```diff
@@ -257,7 +360,7 @@ class Namespaced::Model < ActiveRecord::Base
       id
     end
     searchableAttributes ['customAttr']
-    tags ['static_tag1', 'static_tag2']
+    # tags ['static_tag1', 'static_tag2']
```
Comment (curquiza) [inline spec/integration_spec.rb:363]:
```suggestion
```

File: spec/integration_spec.rb [hunk 3]
```diff
@@ -518,9 +621,9 @@ class SerializedObject < ActiveRecord::Base
     meilisearch :index_name => safe_index_name('SerializedObject') do
       use_serializer SerializedObjectSerializer
 
-      tags do
-        ['tag1', 'tag2']
-      end
+      # tags do
+      #   ['tag1', 'tag2']
+      # end
```
Comment (curquiza) [inline spec/integration_spec.rb:626]:
```suggestion
```

File: spec/integration_spec.rb [hunk 4]
```diff
@@ -299,44 +402,44 @@ def nb_children
 
 # create this index before the class actually loads, to ensure the rankingRules are updated
 # index = MeiliSearch::Index.new(safe_index_name('City_replica2'))
-index = MeiliSearch.client.create_index(safe_index_name('City_replica2'))
+# index = MeiliSearch.client.create_index(safe_index_name('City_replica2'))
```
Comment (curquiza) [inline spec/integration_spec.rb:405]: you can delete all this comments

**Review by curquiza — CHANGES_REQUESTED** [review]
thanks for all these tests caro ❤️

File: spec/integration_spec.rb (hunk 4 above)
Comment (CaroFG) [reply spec/integration_spec.rb:405]: Ok, I was planning to do it for the "remove useless code" issue, but I'll do it now :)

File: lib/meilisearch-rails.rb (hunk 1 above)
Comment (CaroFG) [reply lib/meilisearch-rails.rb:890]: `primary_key `should go before `id` since it's the terminology used by MeiliSearch

**Review by curquiza — APPROVED** [review]
merge when you want :)

**Comment by CaroFG** [comment]
bors merge

**Pull request merged**